tests/periph_pdm/larger_buff_size.py

```diff
@@ -0,0 +1,117 @@
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import re
+import numpy as np
+import os
+import csv
+from itertools import groupby
+from operator import itemgetter
+import serial
+import struct
+import wave
+import time
```

Comment:
If you are providing a script, maybe it is better to provide it in a `tests` sub-folder or maybe even somewhere in `tools/dist`?

It should also have some sort of `requirements.txt` so others can easily setup the and run it.

Reply:
Thank you Kevin for your guidance and assistance.
I will move the python script to the dist/tools folder and include the requirements.txt and readme.md files as well.